### PR TITLE
Add a generic GetUnclaimedGas() in RpcClient

### LINKED ISSUE #269: RpcClient need modification for getwalletbalance and getwalletunclaimedgas
RpcClient need modification since method names were changed here (`getbalance` -> `getwalletbalance`, `getunclaimedgas` -> `getwalletunclaimedgas` )  from https://github.com/neo-project/neo-modules/pull/243

### PR DESCRIPTION
**Pull request by joeqian10**
Close #269

Need to update RPC documentation.

### CODE REVIEW

File: tests/Neo.Network.RPC.Tests/RpcTestCases.json [hunk 1]
```diff
@@ -849,22 +866,6 @@
...
-    },
-    "Response": {
-      "jsonrpc": "2.0",
-      "id": 1,
-      "result": {
-        "balance": "3001101329992600"
-      }
-    }
-  },
   // mock decimals
```
Comment (Tommo-L) [inline tests/Neo.Network.RPC.Tests/RpcTestCases.json:869]:
```suggestion
```

File: src/RpcClient/RpcClient.cs [hunk 2]
```diff
@@ -340,6 +340,11 @@ public RpcInvokeResult InvokeScript(byte[] script, params UInt160[] scriptHashes
             return RpcInvokeResult.FromJson(RpcSend("invokescript", parameters.ToArray()));
         }
 
+        public RpcUnclaimedGas GetUnclaimedGas(string address)
+        {
+            return RpcUnclaimedGas.FromJson(RpcSend("getunclaimedgas", address));
```
Comment (shargon) [inline src/RpcClient/RpcClient.cs:345]: Wait for https://github.com/neo-project/neo-modules/issues/261

Comment (joeqian10) [reply src/RpcClient/RpcClient.cs:345]: I will fix #261 ASAP.

Comment (shargon) [reply src/RpcClient/RpcClient.cs:345]: `get_unclaimed_gas`

Comment (vncoelho) [reply src/RpcClient/RpcClient.cs:345]: Shargon,we just created another PR that is removing _ on the core now.

Comment (shargon) [reply src/RpcClient/RpcClient.cs:345]: I see, but I saw this too https://github.com/neo-project/neo-modules/issues/261#issuecomment-650728723 hahahaha We need to unify criterias

**Pull request merged**